environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentStatus.java

```diff
@@ -72,7 +72,18 @@
     LOAD_BALANCER_ENV_UPDATE_STARTED("Starting load balancer update for environment"),
     LOAD_BALANCER_ENV_UPDATE_FAILED("Failed to update environment with load balancer"),
     LOAD_BALANCER_STACK_UPDATE_STARTED("Starting load balancer update for all data lakes and data hubs"),
-    LOAD_BALANCER_STACK_UPDATE_FAILED("Failed to update data lakes and data hubs with load balancer");
+    LOAD_BALANCER_STACK_UPDATE_FAILED("Failed to update data lakes and data hubs with load balancer"),
+
+    UPGRADE_CCM_VALIDATION_IN_PROGRESS("Validation for Upgrade CCM started"),
+    UPGRADE_CCM_VALIDATION_FAILED("Validation for Upgrade CCM failed"),
+    UPGRADE_CCM_ON_FREEIPA_IN_PROGRESS("Upgrading CCM on Free IPA"),
+    UPGRADE_CCM_ON_FREEIPA_FAILED("Upgrading CCM on Free IPA failed"),
+    UPGRADE_CCM_ON_DATALAKE_IN_PROGRESS("Upgrading CCM on Data Lake"),
+    UPGRADE_CCM_ON_DATALAKE_FAILED("Upgrading CCM on Data Lake failed"),
+    UPGRADE_CCM_ON_DATAHUB_IN_PROGRESS("Upgrading CCM on Data Hub clusters"),
+    UPGRADE_CCM_ON_DATAHUB_FAILED("Upgrading CCM on Data Hub clusters failed"),
+    UPGRADE_CCM_FAILED("Upgrade CCM failed"),
+    UPGRADE_CCM_ROLLING_BACK("Rolling back failed CCM upgrade");
```

Comment:
Would it make sense to sync these messages with their counterparts in `messages.properties`?

Reply:
sure, on the other hand I do not see the purpose of this redundancy :)

Reply:
Agreed, I do not see it either. :smile: